data/txHandler.go

```diff
@@ -146,6 +154,7 @@ func (handler *TxHandler) backlogWorker() {
 				return
 			}
 			if handler.checkAlreadyCommitted(wi) {
+				transactionMessagesDupPreBacklog.Inc(nil)
```

Comment:
One note here: `checkAlreadyCommitted` will return `true` for reasons other than having a duplicate transaction. For example, if `handler.txPool.Test` returns error because of different errors in `TestTransactionGroup`. 
Also, if the transaction last valid is expired, might get false here.

Reply:
I renamed into AlreadyCommitted